templates/patterns/card/pattern-card--variant-search.html.twig

```diff
@@ -33,8 +33,8 @@
 {% include '@oe-bcl/card' with {
   'horizontal': true,
   'horizontal_grid': {
-    'left_col_classes': 'col-md-3 col-lg-2 rounded mw-listing-img',
```

Comment:
unfortunately this now changes the size of the left column in the card, and with it the image and date widget, we need to discuss in the team if this is a breaking change or not.